README.md

```diff
@@ -69,13 +83,65 @@ Then, follow these steps for creating a contribution:
 
 This add-on is distributed under Apache License 2.0. For license terms, see LICENSE.txt.
 
-TEMPLATE_ADDON is written by Flowing Code S.A.
+DateTimeRangePicker is written by Flowing Code S.A.
```

Comment:
FlowingCode/TechnicalDocs#91
https://github.com/FlowingCode/TechnicalDocs/tree/master/addons#naming-convention

Already mentioned in https://github.com/FlowingCode/DateTimeRangePicker/pull/1#discussion_r2091703389